modules/distr-tracing-tempo-config-receiver-tls-for-tempostack.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls-for-tempostack_{context}"]
+= Receiver TLS configuration for a TempoStack instance
+
+You can provide a TLS certificate in a secret or use the service serving certificates that are generated by the {product-title}.
+
+* To provide a TLS certificate in a secret, configure it in the `TempoStack` custom resource.
++
+[NOTE]
+====
+This feature is not supported with the enabled Tempo Gateway.
+====
++
+.TLS for receivers and using a user-provided certificate in a secret
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind:  TempoStack
+# ...
+spec:
+# ...
+  template:
+    distributor:
+      tls:
+        enabled: true # <1>
+        certName: <tls_secret> # <2>
+        caName: <ca_name> # <3>
+# ...
+----
+<1> TLS enabled at the Tempo Distributor.
+<2> Secret containing a `tls.key` key and `tls.crt` certificate that you apply in advance.
+<3> Optional: CA in a config map to enable mutual TLS authentication (mTLS).
+
+* Alternatively, you can use the service serving certificates that are generated by the {product-title}.
++
+[NOTE]
+====
+Mutual TLS authentication (mTLS) is not supported with this feature.
+====
++
+.TLS for receivers and using OpenShift's service serving certificates
```

Comment:
Thank you for spotting this because I missed this instance.
I've just corrected this to `.TLS for receivers and using service serving certificates that are generated by {product-title}`.